build-system/tasks/check-links.js

```diff
@@ -166,13 +166,18 @@ function filterWhitelistedLinks(markdown) {
  * @param {string} markdownFile Path of markdown file, relative to src root.
  * @return {Promise} Used to wait until the async link checker is done.
  */
-function runLinkChecker(markdownFile) {
+async function runLinkChecker(markdownFile) {
+  // The extension generator markdown template has placeholder links that can't
+  // be checked.
+  if (/doc-template\.md$/.test(markdownFile)) {
+    return;
+  }
   // Skip files that were deleted by the PR.
   if (!fs.existsSync(markdownFile)) {
-    return Promise.resolve();
+    return;
```

Comment:
If we're changing the method signature, let's change the JSDoc too; I believe functions declared with async get standard JSDoc, not a promise return (but double check me on that)

Reply:
(No longer valid, see below)